test/create.test.ts

```diff
@@ -28,7 +31,7 @@ describe("commands/create", function () {
   });
 
   test("throws if chain not provided", async function () {
-    const [account] = getAccounts();
+    const [account] = accounts;
```

Comment:
Each time `getAccounts()` is called we create 25 new Wallet instances, we might as well save memory here.